lib/screens/onboarding/login/login_screen.dart

```diff
@@ -96,12 +97,20 @@ class LoginScreen extends StatelessWidget {
                 Row(
                   mainAxisAlignment: MainAxisAlignment.end,
                   children: <Widget>[
-                    Text('Forgot password?',
-                        style: GoogleFonts.poppins(
-                            textStyle: TextStyle(
-                                fontSize: 12,
-                                color: AppColors.buttonShade1,
-                                fontWeight: FontWeight.bold))),
+                  FlatButton (
```

Comment:
Instead of using a FlatButton here, wrap the Forgot Password text with GestureDetector Widget